client/layout/index.jsx

```diff
@@ -155,7 +156,9 @@ const Layout = createReactClass( {
 				{ 'development' === process.env.NODE_ENV && (
 					<AsyncLoad require="components/webpack-build-monitor" placeholder={ null } />
 				) }
-				<InlineHelp />
+				{ ( 'jetpack-connect' !== this.props.section.name ||
+					this.props.currentRoute === '/jetpack/new' ) &&
```

Comment:
I wish these conditionals has better breaks regarding the parens and different conditions 😕 

```js
( notInSection || isTheExceptionRout ) &&
isNotJetpackLogin &&
<Component />
```

Maybe I'll go bug the prettier folks 😁 


The logic seems right 👍

Reply:
Me too. Actually, this is exactly how I had written it before Prettier gets involved.

🤷‍♂️

Reply:
I might be having a dumb moment here (the odds are high 😉  ), so just let me know if so...

If `this.props.currentRoute === '/jetpack/new' ` is true, would that not infer that `this.props.currentRoute !== '/log-in/jetpack'` is also true by default, rendering it tautological?

As in, if the current route is `'/jetpack/new'` it therefore cannot be anything else (`'/log-in/jetpack'` for example), making the check for `this.props.currentRoute !== '/log-in/jetpack'` redundant?

Reply:
> Moments later, he realised that he was in fact having a dumb moment...

Dang brackets! Ignore all of that 😆